### ISSUE TITLE
Report Generation Issues with FPR

### ISSUE THREAD

**Issue by bbloo** [pyGSTio/pyGSTi#31]
I believe there have been two errors associated with report generation and FPR. One I believe was already spotted by Erik, when using global fiducial pair reduction.  

When trying to create a report on master with per-germ fiducial pair reduction there is a similar bug in:
report/results.py at Line 905
```
            elif isinstance(fidPairs,dict) or hasattr(fidPairs,"keys"):
                #Assume fidPairs is a dict indexed by germ
                fidpair_filters = { (x,y): fidPairs[germ] 
                                    for x in Ls[st:] for y in germs }
```
I believe that should be fidPairs[y].

The same error shows up at line 741.

**Comment by bbloo** [comment]
Also line 1474 in the same file.

**Comment by enielse** [comment]
Thanks for catching and notifying us of these bugs.  We've fixed them as of **pyGSTi version 0.9.4**, where FPR should work properly.
